compatibility-tests/std-backtrace/tests/backtrace.rs

```diff
@@ -0,0 +1,76 @@
+#![feature(backtrace)]
```

Comment:
These tests are just copied and pasted from `tests/`. Tests specific to snafu with `std::backtrace::Backtrace` support may be useful.